cmd/gosky/admin.go

```diff
@@ -623,25 +627,25 @@ var takeDownAccountCmd = &cli.Command{
 
 		for _, did := range cctx.Args().Slice() {
 			if !strings.HasPrefix(did, "did:") {
-				phr := &api.ProdHandleResolver{}
-				resp, err := phr.ResolveHandleToDid(ctx, did)
+				dir := identity.DefaultDirectory()
+				resp, err := dir.LookupHandle(ctx, syntax.Handle(did))
```

Comment:
feeling happy that these re-writes are basically one-to-one

Reply:
yeah, the only weird bit is i'm not using the parse handle code and am just casting direct to the handle type. Feels like unnecessary overhead to me